cmake/Modules/Findswig.cmake

```diff
@@ -20,12 +20,13 @@ if(NOT SWIG_EXECUTABLE)
       BUILD_IN_SOURCE ON
       BUILD_COMMAND ${MAKE} swig
       TEST_COMMAND "" # remove test step
+      UPDATE_COMMAND "" # remove update step
       )
   ExternalProject_Get_Property(swig_swig source_dir)
 
   # Predefined vars for local installed SWIG
   set(SWIG_EXECUTABLE ${source_dir}/swig)
-  set(SWIG_DIR ${source_dir})
+  set(SWIG_DIR ${source_dir}/share/swig/3.0.12)
```

Comment:
there's `SWIG_VERSION` in my FindSWIG, i think it can be used

Reply:
@bakhtin this variable is set when the package is downloaded and built with external project. Since the downloaded archive is specified manually, we know its' structure and can manually define the path.

But this variable indeed has to be also set when package is found in the system, so this will be fixed.

@l4l can you explain? There is no such variable in this script, but it can be set here.

Reply:
check the case :) it's a system file

Reply:
Thought about system variable, seems it cannot be used